code/game/objects/items/devices/flashlight.dm

```diff
@@ -98,6 +98,29 @@
 	brightness_on = 2
 	w_class = 1
 
+/obj/item/device/flashlight/heavy
+	name = "heavy duty flashlight"
+	desc = "A high-luminosity flashlight for specialist duties."
+	icon_state = "heavyflashlight"
+	item_state = "heavyflashlight"
+	brightness_on = 7
+	w_class = 3
+	matter = list(DEFAULT_WALL_MATERIAL = 100,"glass" = 70)
+	contained_sprite = 1
+
+/obj/item/device/flashlight/maglight
+	name = "maglight"
+	desc = "A very heavy duty flashlight."

```

Comment:
Are you sure you've not mixed these up? The descriptions and the light strengths seem to be at odds